descriptors/ModuleDescriptor-template.json

```diff
@@ -52,6 +52,21 @@
           "methods": [ "POST" ],
           "pathPattern" : "/authn/password/repeatable",
           "permissionsRequired" : [ "login.password.validate" ]
+        },
+        {
+          "methods": [ "POST" ],
+          "pathPattern" : "/authn/password-reset-action",
+          "permissionsRequired" : [ "login.password-reset-action.post" ]
+        },
+        {
+          "methods": [ "GET" ],
+          "pathPattern" : "/authn/password-reset-action/{actionId}",
+          "permissionsRequired" : [ "login.password-reset-action.get" ]
+        },
+        {
+          "methods": [ "POST" ],
+          "pathPattern" : "/authn/reset-password",
+          "permissionsRequired" : [ "login.password-reset-action.post" ]
```

Comment:
wrong permission name

Reply:
should be smth like login.password-reset.post

Reply:
Thanks!
Done